app/models/shopify_import/product.rb

```diff
@@ -0,0 +1,4 @@
+module ShopifyImport
+  class Product < ShopifyImport::Base
```

Comment:
This is what I'm not a fan of. I don't think it's necessary to create these subclasses. What is the functionality that this subclass provides? Except if you're using it as a `self.class.name` or similar, for which I think there should be a better way of doing it.